views/Timeline.py

```diff
@@ -41,19 +44,22 @@ def options(self, request, tag):
     def data(self, request, tag):
         return json.dumps(self.chart_data)
 
- 
+
 class TimelineView(resource.Resource):
     isLeaf = False
     output = ''
+    measure_index = {'PSS': 4, 'RSS': 5, 'USS': 6}
 
-    def __init__(self, db, process_name_filter):
+    def __init__(self, db, process_name_filter, measure):
         resource.Resource.__init__(self)
         self.db = db
         self.process_name_filter = process_name_filter
+        self.measure = measure
+        self.index = self.measure_index[measure]
```

Comment:
Might be nicer to do a `measure.lower()` to make this case-insensitive. Query params should be case insensitive, and lowercase is more normal anyway.